app/src/main/java/org/oppia/app/topic/play/TopicPlayFragmentPresenter.kt

```diff
@@ -59,9 +60,9 @@ class TopicPlayFragmentPresenter @Inject constructor(
 
   private val topicLiveData: LiveData<Topic> by lazy { getTopicList() }
 
-  // TODO(#135): Get this topic-id or get storyList from [StoryFragment].
   private val topicResultLiveData: LiveData<AsyncResult<Topic>> by lazy {
-    topicController.getTopic(TEST_TOPIC_ID_0)
+    topicId = fragment.arguments?.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY) ?: ""
```

Comment:
Ditto here--let's ensure it's not null as a checkNotNull() and set topicId in a lifecycle method rather than a lazy init.

Reply:
Resolved